php_phongo.c

```diff
@@ -1289,6 +1289,7 @@ static bool php_phongo_apply_wc_options_to_uri(mongoc_uri_t *uri, bson_t *option
 	return true;
 } /* }}} */
 
+#ifdef MONGOC_ENABLE_SSL
```

Comment:
OK, found it. It's just that github didn't include the code between lines 1295 and 1398.